org.metaborg.spoofax.shell.core/src/main/java/org/metaborg/spoofax/shell/commands/HelpCommand.java

```diff
@@ -67,7 +67,7 @@ public IHook execute(String... args) throws MetaborgException {
             }
 
             return (display) -> display
-                .displayMessage(new StyledText(formathelp(commands)));
+                .visitMessage(new StyledText(formathelp(commands)));
         } catch (CommandNotFoundException e) {
             throw new MetaborgException("Command not found: " + e.commandName());
```

Comment:
Should this be a `visitThrown`, or is this caught and handled elsewhere that I haven't seen yet?